addons/mail/models/mail_thread.py

```diff
@@ -4689,16 +4689,19 @@ def _thread_to_store(self, store: Store, fields, *, request_list=None):
                     ]
                 )
                 thread._message_followers_to_store(store, filter_recipients=True, reset=True)
-            if request_list and "display_name" in request_list:
+            if "display_name" in request_list:
                 res["display_name"] = thread.display_name
-            if request_list and "scheduledMessages" in request_list:
+            if "scheduledMessages" in request_list:
                 res["scheduledMessages"] = Store.Many(self.env['mail.scheduled.message'].search([
                     ['model', '=', self._name], ['res_id', '=', thread.id]
                 ]))
-            if request_list and "suggestedRecipients" in request_list:
+            if "suggestedRecipients" in request_list:
                 res["suggestedRecipients"] = thread._message_get_suggested_recipients(
                     reply_discussion=True, no_create=True,
                 )
+            if "contact_fields" in request_list:
```

Comment:
alphabetical with other conditions (unless it negatively impacts cache/query count)